pkgs/development/lisp-modules-obsolete/quicklisp-to-nix-overrides.nix

```
@@ -4,8 +4,7 @@ let
   skipBuildPhase = x: {
     overrides = y: ((x.overrides y) // { buildPhase = "true"; });
   };
-  multiOverride = l: x: if l == [] then {} else
-    ((builtins.head l) x) // (multiOverride (builtins.tail l) x);
+  multiOverride = l: x: lib.optionalAttrs (l != []) ((builtins.head l) x) // (multiOverride (builtins.tail l) x);
```

`lib` isn't in scope here.
```suggestion
  multiOverride = l: x: pkgs.lib.optionalAttrs (l != []) ((builtins.head l) x) // (multiOverride (builtins.tail l) x);
```